src/winrpath.cxx

```diff
@@ -496,6 +501,58 @@ void CoffParser::ParseData(PIMAGE_ARCHIVE_MEMBER_HEADER header, coff_member *mem
 }
 
 
+void CoffParser::NormalizeLinkerMember(
```

Comment:
(question) compared to NormalizeSectionNames (which appears to fix some bugs) this looks more like pure consolidation - did I get that right?

Reply:
Yes, this is a pure code reorganization. `NormalizeName` was too long and messy IMO.